collect_app/src/main/java/org/odk/collect/android/utilities/IconUtils.java

```diff
@@ -29,4 +34,15 @@ public static int getNotificationAppIcon() {
         return Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP ?
                 R.drawable.ic_notes_white : R.drawable.ic_notes_white_png;
     }
+
+    /** Renders a Drawable (such as a vector drawable) into a Bitmap. */
+    public static Bitmap getBitmap(Context context, int drawableId) {
```

Comment:
Since others may use this method in the future, can you add the optimizations from [this SO answer](https://stackoverflow.com/a/10600736/763231)? Overall very similar, but checks a few edge cases.

Reply:
Yep!  Added handling for bitmaps and solid colours.